back/routes/adventure.js

```diff
@@ -67,7 +67,7 @@ async function getDatabaseRestaurants(distance, userLat, userLng) {
   // IDs that are not allowed to be repeated by google API
   const usedPlaceIDs = [];
   for (const restaurant of allRestaurants) {
-    if (!restaurant.address) continue;
+    if (!restaurant.address || restaurant.activeFeedback.length < 5) continue;
```

Comment:
This looks a bit like a [magic number](https://en.wikipedia.org/wiki/Magic_number_(programming)#:~:text=The%20term%20magic%20number%20or,1%20manuals%20of%20the%201960s.) - maybe we can declare it as a constant?

Reply:
I will! Thanks for that!